online-shop/src/components/product/product.component.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+// import { preProcessFile } from 'typescript';
+
+import "./product.styles.css"
+
+
+export interface Produs {
+    id:number,
+    name:String,
+    category:String,
+    price: number,
+    imageUrl: String
+}
+
+// m-am oprit aici cu style ul pana ma uit peste materialUI :(
+// nu e responsive inca
+
+const Product = ( props:Produs ) =>(
+    <div className="product">
+        <div className="headProd">
+  
+            <h1 className="prodName">{props.name}</h1>
+            <h1 className="prodcateogry">({ props.category})</h1>
+    
```

Comment:
Again, remove all the empty lines because they are only increasing the lenght of the files